tools/sifgen/common/types/genesis.go

```diff
@@ -175,6 +175,20 @@ type StakingParams struct {
 	BondDenom         string `json:"bond_denom"`
 }
 
+type Mint struct {
+	Minter interface{} `json:"minter" yaml:"minter"`
+	Params MintParams  `json:"params" yaml:"params"`
+}
+
+type MintParams struct {
```

Comment:
Are the default values okay ? 
MintDenom:           sdk.DefaultBondDenom,
InflationRateChange: sdk.NewDecWithPrec(13, 2),
InflationMax:        sdk.NewDecWithPrec(20, 2),
InflationMin:        sdk.NewDecWithPrec(7, 2),
GoalBonded:          sdk.NewDecWithPrec(67, 2),
BlocksPerYear:       uint64(60 * 60 * 8766 / 5), // assuming 5 second block times

If not , we might need to add custom values to tools -> sifgen - > genesis.json

Reply:
Yeah, the goal right now is to just bring in stock cosmos. We'll be modifying down the road when we implement rebalancing.